### ISSUE TITLE
Snowflake transient table type option

### ISSUE THREAD

**Issue by nugic** [transferwise/pipelinewise#592]
By default tables in Snowflake are created as of permanent type. That means that a time travel period set in account parameters and a 7 day fail safe period apply to this table. When using WAL based data sync with lot of updates, this setting could mean quite high storage growth because of time travel and fail safe implications. If these kind of backup options are not actually needed, then the solution would be to use transient type tables, which have no fail safe and up to 1 day time travel period (in some cases it would be acceptable just to do the initial full sync again in case of a failure). 

Proposing to add an option to Pipelinewise target_snowflake if created schema or tables are permanent (as default) or transient type.

**Comment by Samira-El** [comment]
Hey,

This is a great idea and feature to have, especially costs-wise!

We could have both the option of flagging a schema transient, thus all its tables are transients as well, and the option of flagging only certain tables as transient in a permanent schema.

**Comment by nauxliu** [comment]
Any updates on this? We have a particularly active table that updates very frequently, which leads to time travel and fail-safe consumption that is several orders of magnitude greater than active storage, resulting in the use of about 20T of storage in places where it is not needed.

**Comment by nugic** [comment]
@nauxliu I don't think it is implemented yet. We have solved it so, that the whole schema, where pipelinewise imports tables, is created as transient, meaning that all tables created in the schema are transient also.
